aspnetcore-webapi/BuildQueryString/BuildQueryString/HttpClientWrapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace BuildQueryString
+{
+    public class HttpClientWrapper : IHttpClientWrapper
+    {
+        public async Task<HttpResponseMessage> GetAsync(string requestUri)
+        {
+            using HttpClient _httpClient = new();
+
+            return await _httpClient.GetAsync(requestUri);
```

Comment:
This block of code is presenting a very bad practice in how the HttpClient is being used. Here is one article discussing it, but in short it can lead to socket exhaustion:
https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

Reply:
Thank you for your insights. I've removed the using block.